sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSourceSuite.scala

```diff
@@ -1777,6 +1817,19 @@ class FileStreamSourceSuite extends FileStreamSourceTest {
     }
   }
 
+
+  test("SPARK-19813: Ignore maxFileAge when maxBytesPerTrigger and latestFirst is used") {
```

Comment:
`maxFilesPerTrigger` was removed from the test title. We should keep it as before.

Reply:
Oh, nvm, just found that this is a new test added.